pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -350,16 +353,22 @@ func getPromExemplars(pt pdata.HistogramDataPoint) []prompb.Exemplar {
 			Timestamp: timestamp.FromTime(exemplar.Timestamp().AsTime()),
 		}
 
+		exemplarChars := 0
 		exemplar.FilteredAttributes().Range(func(key string, value pdata.AttributeValue) bool {
+			val := value.AsString()
+			exemplarChars += len(key) + len(val)
```

Comment:
Being very nitpicky here but: `len` returns the number of bytes, not UTF-8 code points (see e.g. https://go.dev/play/p/_9lpGqY2gkc from https://pkg.go.dev/unicode/utf8#RuneCountInString), so at least for `val` you need to use `utf8.RuneCountInString`

Reply:
Good catch.  This is probably safe since bytes must be >= runes, but would be good to not drop data unnecessarily.

Reply:
done